README.rst

```diff
@@ -5,103 +5,67 @@
-    cd scikit-bio
-    pip install .
-
-After this completes, you can run the scikit-bio unit tests from with a Python or IPython session as follows::
-
-    import skbio
-    skbio.test()
-
-To only run tests for a specific scikit-bio module, for example skbio.sequence, you can do::
-
-    skbio.sequence.test()
-
-For developers of scikit-bio, if you don't want to be forced to re-install after every change, you can modify the above ``pip install`` command to::
-
-    pip install -e .
-
-This will build scikit-bio's Cython extensions, and will create a link in the ``site-packages`` directory to the scikit-bio source directory. When you then make changes to code in the source directory, those will be used (e.g., by the unit tests) without re-installing.
-
-Finally, if you don't want to use ``pip`` to install scikit-bio, and prefer to just put ``scikit-bio`` in your ``$PYTHONPATH``, at the minimum you should run::
-
-    python setup.py build_ext --inplace
-
-This will build scikit-bio's Cython extensions, but not create a link to the scikit-bio source directory in ``site-packages``. If this isn't done, using certain components of scikit-bio will be inefficient and will produce an ``EfficiencyWarning``.
-
 Getting help
 ------------
 
-To get help with scikit-bio, you should use the `skbio <http://stackoverflow.com/questions/tagged/skbio>`_ tag on StackOverflow (SO). Before posting a question, check out SO's guide on how to `ask a question <http://stackoverflow.com/questions/how-to-ask>`_. The scikit-bio developers regularly monitor the skbio SO tag.
-
-Licensing
----------
-
-scikit-bio is available under the new BSD license. See
-`COPYING.txt <https://github.com/biocore/scikit-bio/blob/master/COPYING.txt>`__ for scikit-bio's license, and the
-`licenses directory <https://github.com/biocore/scikit-bio/tree/master/licenses>`_ for the licenses of third-party software that is
-(either partially or entirely) distributed with scikit-bio.
+To get help with scikit-bio, you should use the `skbio <http://stackoverflow.com/questions/tagged/skbio>`_ tag on StackOverflow (SO). Before posting a question, check out SO's guide on how to `ask a question <http://stackoverflow.com/questions/how-to-ask>`_. The scikit-bio developers regularly monitor the ``skbio`` SO tag.
 
 Projects using scikit-bio
 -------------------------
 
 Some of the projects that we know of that are using scikit-bio are:
 
--  `QIIME <http://qiime.org/>`__
--  `Emperor <http://biocore.github.io/emperor/>`__
--  `An Introduction to Applied
-   Bioinformatics <http://caporasolab.us/An-Introduction-To-Applied-Bioinformatics/>`__
--  `tax2tree <https://github.com/biocore/tax2tree>`__
+- `QIIME <http://qiime.org/>`__
+- `Emperor <http://biocore.github.io/emperor/>`__
+- `An Introduction to Applied
+  Bioinformatics <http://readIAB.org>`__
+- `tax2tree <https://github.com/biocore/tax2tree>`__
+- `Qiita <http://qiita.microbio.me>`__
+- `ghost-tree <https://github.com/JTFouquier/ghost-tree>`__
```

Comment:
Although rather recent, Platypus-Conquistador is also making use:
https://github.com/biocore/Platypus-Conquistador